docs/api/service-workers.md

```diff
@@ -64,10 +75,56 @@ The following methods are available on instances of `ServiceWorkers`:
 
 Returns `Record<number, ServiceWorkerInfo>` - A [ServiceWorkerInfo](structures/service-worker-info.md) object where the keys are the service worker version ID and the values are the information about that service worker.
 
-#### `serviceWorkers.getFromVersionID(versionId)`
+#### `serviceWorkers.getInfoFromVersionID(versionId)`
+
+* `versionId` number - ID of the service worker version
 
-* `versionId` number
+Returns [`ServiceWorkerInfo`](structures/service-worker-info.md) - Information about this service worker
+
+If the service worker does not exist or is not running this method will throw an exception.
+
+#### `serviceWorkers.getFromVersionID(versionId)` _Deprecated_
+
+* `versionId` number - ID of the service worker version
 
 Returns [`ServiceWorkerInfo`](structures/service-worker-info.md) - Information about this service worker
 
 If the service worker does not exist or is not running this method will throw an exception.
+
+**Deprecated:** Use the new `serviceWorkers.getInfoFromVersionID` API.
+
+#### `serviceWorkers.getWorkerFromVersionID(versionId)` _Experimental_
+
+* `versionId` number - ID of the service worker version
+
+Returns [`ServiceWorkerMain | undefined`](service-worker-main.md) - Instance of the service worker associated with the given version ID.
```

Comment:
If I understand correctly, when a ServiceWorker is cleaned up by Chromium, this API could return `undefined`, but that happens at some indeterminate point after the worker becomes "redundant", or possibly in other cases?

Is it possible to explain any of that info here without having to fully explain the lifecycle of ServiceWorkers? If so, it might help people understand how to handle `undefined` return values.

Perhaps adding something like:

* "Returns `undefined` if the ServiceWorker has been cleaned up"
* "Returns `undefined` if the ServiceWorker no longer exists"
* "Returns `undefined` if the ServiceWorker has been unregistered"

I'm not sure which terminology is the most accurate way to describe this, so I'm offering a few suggestions in case any of them sound reasonable 😅

Reply:
Clarified that it relates to the service worker's running status in [96dedfd](https://github.com/electron/electron/pull/45232/commits/96dedfd3ce0280486e2f824b599ea4c3f567a96f)